test_util/test_runner.py

```diff
@@ -19,7 +19,7 @@ def integration_test(
         dcos_dns, master_list, agent_list, public_agent_list,
         test_dns_search, provider,
         aws_access_key_id='', aws_secret_access_key='', region='', add_env=None,
-        pytest_cmd='py.test -vv',
+        pytest_cmd='py.test -rs -vv',
```

Comment:
I'd like to land it independently / before all of this lands, hence separate PR (can leave the patch in this PR as well and after it lands git rebase / merge will fix things up).